hack/e2e-scripts/aws-e2e-pr-gather.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Export and copy api-server logs back to the runner
+  hosts: localhost
+  gather_facts: true
+  tasks:
+    - name: Verify Connectivity from a spoke node to all spokes
```

Comment:
i might be missing here something, but the name of the task doesn't match with the shell command. ignore if it's correct.